pages/index.js

```diff
@@ -116,7 +116,7 @@ function HomePage() {
                   </Paragraph>
                 </div>
                 <div className="mt-5 flex">
```

Comment:
For justify button for >md views.

```suggestion
                <div className="mt-5 flex justify-center">
```

Reply:
Oh okay, will effect the change @magicmatatjahu

Reply:
Hi @magicmatatjahu, With respect to the lock file being reverted how do I go about that since it's been committed?

Thanks.

Reply:
You need to revert the given commit, or more simply, copy the content from https://github.com/asyncapi/website/blob/master/package-lock.json and paste it into your package-lock.

Reply:
Oh okay

Reply:
@magicmatatjahu, just updated the PR

Reply:
@magicmatatjahu is that the goal for the Join us button >md screen

<img width="1437" alt="Screenshot 2022-05-10 at 12 19 27 PM" src="https://user-images.githubusercontent.com/26670804/167617477-e80fabc1-b47b-4b17-b7bb-c80dc22a6775.png">